### DISCUSSION TITLE
Apply installed extensions to Zen Sidebar

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#777] [Ideas]
### What feature would you like to see?

I would like to see my installed extensions applied to the Zen Sidebar.
The reason is: I have the "Dark Reader" extension installed, which makes every website appear in dark mode.
Without it, my eyes burns. Thanks!

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/645_

**Comment by RobotoSkunk** [comment]
I think that more than a feature, this is a bug. Some extensions are applied to Zen Sidebar, but some other doesn't
